test/tstDBSCAN.cpp

```diff
@@ -114,6 +122,27 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(dbscan, DeviceType, ARBORX_DEVICE_TYPES)
   using ArborX::Point;
   using ArborX::Details::verifyDBSCAN;
 
+  {
+    // Disable this test for CUDA, as it results in warning about calling
+    // __host__ function from __host__ __device__ one.
+#ifndef KOKKOS_ENABLE_CUDA
```

Comment:
Use a view and relax this

Reply:
Just to make sure: are you proposing to use view, but make it a separate class, and specialize `AccessTraits` on it? The point here is to not use `View` for primitives.

Reply:
Put it in a class if you like

Reply:
Done.